DailyCodingProblem/DailyCodingProblem/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DailyCodingProblem
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Problem1();
+        }
+
+        static void Problem1()
+        {
+            // Given a list of numbers and a number k, return whether any two numbers from the list add up to k.           
+
+            List<int> listOfNumbers = new List<int>() { 8, 1, 9, 3, 7, 6, 12, 36, 78, 25, 5 };
+
+            Console.Write($"Problem 1:\nPlease enter a number and I'll tell you if any two numbers from the list add up to it: ");
+
+            int number;
+
+            while(!int.TryParse(Console.ReadLine(), out number))
+            {
+                Console.Write("Your input is invalid. Please enter a valid number: ");
+            }
+
+            for(int i = 0; i < listOfNumbers.Count; i ++)
```

Comment:
Seems to be some minor spacing weirdness, not a big deal but would suggest doing a Control + K + D